docs/docs/kafka-connect.md

```diff
@@ -0,0 +1,352 @@
+App Registration. You might have to choose "Client secrets" in the middle panel and the "+" in front of "New client secret"
+to generate one. Be sure to set this variable to the Value and not the Id.
+
+It's also important that the App Registration is granted the Role Assignment "Storage Blob Data Contributor" in your
+Storage Account's Access Control (IAM), or it won't be able to write new files there.
+
+Then, within the Connector's configuration, you'll want to include the following:
+
+```
+"iceberg.catalog.type": "rest",
+"iceberg.catalog.uri": "https://catalog:8181",
+"iceberg.catalog.warehouse": "abfss://storage-container-name@storageaccount.dfs.core.windows.net/warehouse",
+"iceberg.catalog.io-impl": "org.apache.iceberg.azure.adlsv2.ADLSFileIO",
+"iceberg.catalog.include-credentials": "true"
+```
+
+Where `storage-container-name` is the container name within your Azure Storage Account, `/warehouse` is the location
+within that container where your Apache Iceberg files will be written by default (or if iceberg.tables.auto-create-enabled=true),
+and the `include-credentials` parameter passes along the Azure Java client credentials along. This will configure the
+Iceberg Sink connector to connect to the REST catalog implementation at `iceberg.catalog.uri` to obtain the required
+Connection String for the ADLSv2 client
+
+### Google GCS configuration example
+
+By default, Application Default Credentials (ADC) will be used to connect to GCS. Details on how ADC works can
+be found in the [Google Cloud documentation](https://cloud.google.com/docs/authentication/application-default-credentials).
+
+```
+"iceberg.catalog.type": "rest",
+"iceberg.catalog.uri": "https://catalog:8181",
+"iceberg.catalog.warehouse": "gs://bucket-name/warehouse",
+"iceberg.catalog.io-impl": "org.apache.iceberg.google.gcs.GCSFileIO"
+```
+
+### Hadoop configuration
+
+When using HDFS or Hive, the sink will initialize the Hadoop configuration. First, config files
+from the classpath are loaded. Next, if `iceberg.hadoop-conf-dir` is specified, config files
+are loaded from that location. Finally, any `iceberg.hadoop.*` properties from the sink config are
+applied. When merging these, the order of precedence is sink config > config dir > classpath.
+
+## Examples
+
+### Initial setup
+
+#### Source topic
+
+This assumes the source topic already exists and is named `events`.
+
+#### Control topic
+
+If your Kafka cluster has `auto.create.topics.enable` set to `true` (the default), then the control topic will be
+automatically created. If not, then you will need to create the topic first. The default topic name is `control-iceberg`:
+```bash
+bin/kafka-topics  \
+  --command-config command-config.props \
+  --bootstrap-server ${CONNECT_BOOTSTRAP_SERVERS} \
+  --create \
+  --topic control-iceberg \
+  --partitions 1
+```
+*NOTE: Clusters running on Confluent Cloud have `auto.create.topics.enable` set to `false` by default.*
```

Comment:
Default value itself is false right? Why this special note about "confluent cloud"?

Reply:
Default is true for plain Kafka AFAIK.

Reply:
Sorry, I got confused with auto create tables. This is about topic. 

LGTM.